src/sanitize.js

```diff
@@ -8,7 +8,10 @@ const clean = dirty => sanitizeHtml(dirty, {
 export const sanitizeElementName = clean;
 export const sanitizeAttrVal = clean;
 export const sanitizeAttr = clean;
-export const sanitizeJSON = dirty => JSON.stringify(dirty).replace(/</g, '\\u003c');
+export const sanitizeJSON = (dirty) => {
+  const result = JSON.stringify(dirty);
```

Comment:
@nicksorrentino `stringify` is extremely fast, in fact aphrodite uses it to generate every single className hash which happens hundreds (or thousands?) of times on our pages. In any case, `stringify` is not doing any sanitization, the following regex is. This function is serving the dual purpose of generating the string that we insert into the markup as well as sanitizing it.

Re: the sanitization: while I agree that it's unlikely that malicious UGC will filter through to this point in the page rendering pipeline, it is probable that some of our json schema values will come from UGC and this is here to safeguard that.